src/base.jl

```diff
@@ -46,8 +46,22 @@ function ShortString{T}(s::ShortString{S}) where {T, S}
     ShortString{T}(content | T(sz))
 end
 
+const shft_int = UInt === UInt32 ? 2 : 3
```

Comment:
Docstring?

Reply:
should constants be all capitalised? Like `SHFT_INT`